subscriptions/configuration/subscription_templates.rst

```diff
@@ -0,0 +1,91 @@
+==========================
+Use subscription templates
+==========================
+
+The subscription business model is becoming more popular. Are you wondering why?
+
+- For **customers**, value lies in **convenience**:
+
+  - Subscriptions simplify the business process. Indeed, subscribers never have to remember to renew
+    their orders every month, which gives them the assurance that they will have everything they
+    need before they actually need it.
+  - Subscriptions help customers stay on budget.
+  - Subscriptions bring added value to the customer.
+
+- For **businesses**, value lies in the **ability to predict recurring revenue**:
+
+  - Subscriptions reduce customer churn rate and significantly increase customer retention.
+  - Subscriptions provide much higher payment security for your business. They stabilize and
+    maintain recurring revenue streams. Indeed, the more guaranteed revenue you can offer, the more
+    valuable your business will be.
+
+.. raw:: html
+
+   <div align="center" style="color:#AD5E99; font-size: 2rem ;margin: 20px 0"> <b>Simplicity.
+   Higher customer retention. Opportunities for marketing. Business consistency. Better cash flow
+   management.</b> </div>
+
+Odoo subscription templates will help you save a lot of time and money. Indeed, these templates will
+help you generate recurring invoices and manage renewals at a fast pace. With Odoo you have the
+possibility to create, edit and manage your own subscription templates.
```

Comment:
```suggestion
Odoo subscription templates help you save time and money. Templates can
help you generate recurring invoices and manage renewals at a fast pace. With Odoo you have the
possibility to create, edit, and manage your own subscription templates.
```

From the book Style, he says : 'don’t modify absolute words'. That is why throughout the docs I review I suggest remove such words (: